3_Development/git-n00bs.md

```diff
@@ -0,0 +1,61 @@
+# Git for Non-Developers
+
+We use git for everything, including documentation and that means non-developers use it too. Since we're not that comfortable with it or using it daily, here's a short guide on the basics. Instructions are for MacOS.
+
+### Add Repository
+
+Open terminal, open the local folder where you want the repository to be. 
+
+Copy the link from GitHub repository Clone or Download green button.
+
+`git clone [git_link]`
+
+### Sync Repository
+
+Run terminal, type git. Go to root directory of the repository on localhost (cd ..)
+
+`git fetch -p`
+
+### Create Branch
+
+Name branch based on the changes you're making. 
+
+`git checkout -B branch_name origin/master`
+
+### Add or Change Files
+
+Changes are automatically tracked but if you created new files, you need to add them:
+
+`git add .`
```

Comment:
Pls don't do this. Adding everything is a good recipe to get into troubles. If you don't want to add a single file at a time you can do `git add -i` (-i is "interactive") and then follow the "wizard".

Reply:
Non-devs rarely do anything but add documentation. I just added a bunch of docs from /support to easyblognetworks/support. Would it make a difference if I went through the wizard?